### PR TITLE
 Add BackgroundSubtractorCNT

### PR DESCRIPTION
**Pull request by makasim**

### CODE REVIEW

File: contrib/bgsegm_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package contrib
+
+import (
+	"testing"
+	"gocv.io/x/gocv"
+	v "gocv.io/x/gocv/contrib"
```
Comment (makasim) [inline contrib/bgsegm_test.go]: The test fails without this line. I've no idea why.

Comment (netbrain) [reply contrib/bgsegm_test.go]: why "v" as package name and not the default "contrib"?

Comment (netbrain) [reply contrib/bgsegm_test.go]: you also need to gofmt your changes.

Comment (acharyab15) [reply contrib/bgsegm_test.go]: I pulled your branch, merged the current `dev` branch and removed the line importing `gocv.io/x/gocv/contrib` (i believe it had some circular imports). Seems like the tests pass now without it.

**Comment by deadprogram** [comment]
Hello, any way to try to revive this PR? From the last CI run would appear that it just needs gofmt run on it:

https://travis-ci.org/hybridgroup/gocv/builds/508827040#L851

Thanks!

**Comment by deadprogram** [comment]
I rebased and ran go fmt on this code, then merged into `dev`.

Thank you @makasim for your contribution. Now closing this PR as the code was merged.

**Pull request closed without merging**